GVRf/Framework/src/org/gearvrf/scene_objects/GVRTextViewSceneObject.java

```diff
@@ -39,7 +39,32 @@
 import android.widget.TextView;
 
 public class GVRTextViewSceneObject extends GVRSceneObject {
```

Comment:
I think finally we can make something like GVRViewGroupSceneObject to let developer to make their customized layout other than textviews' group.